### ISSUE TITLE
Multi language support possible ?

### ISSUE THREAD

**Issue by xiaoouwang** [neilsustc/vscode-dic-completion#1]
Tks for this plugin ! Any chance a dict is possible for other languages like french and spanish ?

**Comment by neilsustc** [comment]
Yes of course, but I need some help because I have little knowledge about those languages. (providing me with a proper word list)

**Comment by xiaoouwang** [comment]
http://lexique.org/public/Lexique382.zip

found one :DDDD

**Comment by xiaoouwang** [comment]
the leftmost one is the words

**Comment by neilsustc** [comment]
Are these words frequently used in French?

![image](https://user-images.githubusercontent.com/7588612/34067017-1eca82b4-e255-11e7-855a-c4539f2b83ac.png)

**Comment by neilsustc** [comment]
That will be great because the `Lexique382` file contains about 140,000 words. I think most of them might not be frequently used and it will cause performance issues in vscode.

For comparison, we have 5,003 words for English now. I think 5,000~20000 would be enough.

**Comment by Jaimer** [comment]
Heres a list of spanish words ordered by frequence of use
[http://sadowsky.cl/files/lifcach/Sadowsky_&_Martinez_-_LIFCACH-2.0.zip](Lista de Frecuencias de Palabras del Castellano de Chile)

**Comment by jakobhviid** [comment]
Hi, interesting feature request. 

For danish (da) we can use this list:
https://github.com/aslak/dansk-diceware/blob/master/dansk-diceware.txt

It is about 60.000 words, but I think it can be reduced a bit if you want me to adapt it.

**Comment by yzhang-gh** [comment]
For now, there is a workaround since we already supported to add custom wordlist.
